.github/workflows/ci.yml

```diff
@@ -244,15 +251,23 @@ jobs:
       matrix:
         include: ${{ fromJson(needs.setup.outputs.test_matrix) }}
     steps:
-      - name: Download zcash-btest artifact
-        uses: actions/download-artifact@v4
+      - name: "Set up Cloud SDK"
+        if: runner.os != 'Linux'
+        uses: "google-github-actions/setup-gcloud@v2"
         with:
-          name: zcash-btest-${{ matrix.name }}
+          version: ">= 363.0.0"
+
+      - name: Download zcash-btest artifact
+        shell: bash
+        run: |
+          ./.github/gcs-download-artifacts.sh ${{ secrets.GCP_SA_KEY }} ${{ github.run_id }}/${{ matrix.name }} ${{ format('src/test/test_bitcoin{0}', matrix.file_ext) }} ./src/test/
```

Comment:
Rather than partially reconstructing the path for the uploaded artifact, we should just use [the output that the other job provides](https://github.com/google-github-actions/upload-cloud-storage?tab=readme-ov-file#outputs). In this case it would be something like `${{ needs.build.steps.upload-btest.outputs.uploaded }}`.

Reply:
I like the idea, I'm going to try how it works

Reply:
It seems that implementing this complicates the code and its potential maintenance.